spec/controllers/blog_posts_controller_spec.rb

```diff
@@ -22,6 +22,12 @@
       get :index
       expect(response).to render_template('blog_posts/index')
     end
+
+    it "paginates" do
```

Comment:
Ah, I didn't realize. (I have never created a feature test, so this will be my first one.)

I made this a feature test inside the ```spec/features``` folder, ```paginate_posts_index_spec.rb```, which seems to be passing, but Kaminari's ```paginate``` method is now throwing an error across other tests:
```
NoMethodError:
     #   undefined method `total_pages' for #<BlogPost::ActiveRecord_Relation:0x007ff16a2a8038>
```

I can work on this.